testing/btest/bifs/sleep.zeek

```diff
@@ -0,0 +1,21 @@
+# Verifies sleep()'s reported latencies.
+#
+# @TEST-EXEC: zeek -b %INPUT 2>out
+# @TEST-EXEC: btest-diff out
```

Comment:
This was mainly to trigger a failure when `assert()` produces output. `assert()` does not error out, right? I felt this was easier than using an `assert_failure()` hook, but let me know if I'm not holding this right.

Reply:
You're right. Because we run with `ZEEK_ALLOW_INIT_ERRORS=1`, failing asserts in `zeek_init()` won't cause Zeek to exit with a failure.

Once `assert` testing is more established, might look for another pattern. If `zeek` would exit with failure, we could just rely on `.stderr` being populated. That's also shown by default when running `btest -d` on failure.